ONSdigital/ssdc-rm-docker-dev
# SDCSRM-529-add-support-frontend

# Motivation and Context
the new Support Frontend needs to be added

# What has changed
Added the new support frontend

# How to test?
build the new frontend, the run `make up` and check you can go to the index page

# Links
[Jira - SDCSRM-529](https://jira.ons.gov.uk/browse/SDCSRM-529)

# Screenshots (if appropriate):
